src/index.js

```diff
@@ -109,12 +117,22 @@ const decorator = ({ declareQueries, declareCommands, declareConnect }) => (Comp
   ]);
 
   @composedDecorators
-  @skinnable(contains(Component))
+  @skinnable(contains(loadingDecorator(Component)))
   @pure
   @props(propsTypes)
   class ContainerFactoryWrapper extends React.Component { // eslint-disable-line react/no-multi-comp
+
     static displayName = displayName(Component, 'Container');
-    getLocals = getLocals;
+
+    getLocals(props) {
+      const { readyState } = props;
+      if (isReady(props)) {
+        return { ...stripUndef({ readyState }), ...getLocals(props) };
+      } else {
+        return stripUndef({ readyState });
+      }
+    }
+
```

Comment:
what happens when no `queries` are defined on the container? Probably it works just fine (no `readyState` passed, `isReady` always return `true`, we always end up in the `then` branch), but maybe everything's a bit too implicit?